### PR TITLE
PRIME-2050 remote users defect in site registration

### PR DESCRIPTION
**Pull request by AMiKaH**
[PRIME-2050](https://bcgovmoh.atlassian.net/jira/software/c/projects/PRIME/boards/1?modal=detail&selectedIssue=PRIME-2050)

**Remote Users now have a single certification instead of a list of certifications.**

### How to test:

1. Create a site
2. Add remote user
3. Click **Back** (you should see the "Unsaved Changes" warning dialogue)
4. Select **Keep Changes**
4. Click **Save and Continue** so you save that new remote user
5. Go back to Remote Users page

###
1. Click **Update Information** on the RU card
2. Click **Back** without changes (no dialogue)
3. In Remote Users page Click **Back** again (You should not see the dialogue)

###
1. Click update **Update Information**
2. Make changes
3. Click **Back**  (you should get the "Unsaved Changes" warning dialogue)
4. Discard Changes
5. **Remote Practitioner added/updated** should not be displayed
6. In Remote Users page Click **Back** again (You should not see the dialogue)

###
1. Click update **Update Information**
2. Make changes
3. Click **Continue** 
4. **Remote Practitioner added/updated** should not be displayed
5. In Remote Users page Click **Back** again (you should see the "Unsaved Changes" warning dialogue)

After all tests are good. Submit site.

Check mailhog and see the **Site Registration Review** document has the proper remote users and their details.

### CODE REVIEW

**Comment by AMiKaH** [comment]
Fixing banner showing up even when no change took place.

File: prime-dotnet-webapi/Models/SiteRegistration/RemoteUser.cs [hunk 1]
```diff
@@ -30,6 +30,6 @@ public class RemoteUser : BaseAuditable
         /// </summary>
         public bool Notified { get; set; }
 
-        public ICollection<RemoteUserCertification> RemoteUserCertifications { get; set; }
+        public RemoteUserCertification RemoteUserCertification { get; set; }
```
Comment (james-hollinger) [inline prime-dotnet-webapi/Models/SiteRegistration/RemoteUser.cs:33]: have you checked with the prod data to see if any existing users have multiple certifications before making this change?

File: prime-dotnet-webapi/ViewModels/SiteRegistration/_SiteMappingProfile.cs [hunk 2]
```diff
@@ -28,7 +28,7 @@ public SiteMappingProfile()
             CreateMap<IndividualDeviceProvider, IndividualDeviceProviderViewModel>();
             CreateMap<IndividualDeviceProviderChangeModel, IndividualDeviceProvider>();
             CreateMap<SiteRemoteUserUpdateModel, RemoteUser>()
-                .ForMember(dest => dest.RemoteUserCertifications, opt => opt.MapFrom(src => new[] { src.RemoteUserCertification }));
+                .ForMember(dest => dest.RemoteUserCertification, opt => opt.MapFrom(src => src.RemoteUserCertification));
```
Comment (james-hollinger) [inline prime-dotnet-webapi/ViewModels/SiteRegistration/_SiteMappingProfile.cs:31]: This mapping is not needed

File: prime-dotnet-webapi/Models/SiteRegistration/RemoteUser.cs (hunk 1 above)
Comment (AMiKaH) [reply prime-dotnet-webapi/Models/SiteRegistration/RemoteUser.cs:33]: Yes. There are no remote users with multiple certifications

**Review by Paahn — APPROVED** [review]
LGTM

File: prime-angular-frontend/src/app/lib/models/remote-user.model.ts [hunk 3]
```diff
@@ -8,5 +8,4 @@ export interface RemoteUser {
   remoteUserCertification: RemoteUserCertification;
   notified: boolean;
   // TODO: community site view model needed
```
Comment (hogmann) [inline prime-angular-frontend/src/app/lib/models/remote-user.model.ts:10]: This comment is now irrelevant, and can be removed.

**Review by hogmann — APPROVED** [review]
LGTM - just comment needs to be removed

**Pull request merged**